### PR TITLE
Clearing warnings and errors from the build command.

### PR DESCRIPTION
**Pull request by Ry-Hebert**
### Not sure if this is the method i should be using to submit updates so let me know if i should do something else.

@electrovir and i talked about clearing the warnings and errors that get thrown when trying to run the build command. Here are a few files that have bene updated to clear build warnings.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -19,3 +19,6 @@
 npm-debug.log*
 yarn-debug.log*
 yarn-error.log*
+
+# dev
+/build
```
Comment (electrovir) [inline .gitignore:24]: #15 handles removing the `build` directory from git so this PR shouldn't make this change.

- [ ] don't git-ignore `build` in this PR

File: src/App.js [hunk 2]
```diff
@@ -231,7 +231,7 @@ export default function App() {
           listing.price,
           10000
         );
-      var r3;
+      // var r3; // Unused var
```
Comment (electrovir) [inline src/App.js:234]: These unused vars line can simply be deleted.

- [ ] delete unused vars

File: src/App.js [hunk 3]
```diff
@@ -674,7 +674,7 @@ export default function App() {
   const footer = (
   <div className={classes.footer}>
     <Typography variant="body1">
-      Developed by ToniqLabs &copy; All rights reserved 2021<br /><a href="https://docs.google.com/document/d/13aj8of_UXdByGoFdMEbbIyltXMn0TXHiUie2jO-qnNk/edit" target="_blank">Terms of Service</a>
+      Developed by ToniqLabs &copy; All rights reserved 2021<br /><a href="https://docs.google.com/document/d/13aj8of_UXdByGoFdMEbbIyltXMn0TXHiUie2jO-qnNk/edit" target="_blank" rel="noreferrer">Terms of Service</a>
```
Comment (electrovir) [inline src/App.js:677]: Let's add `noopener` to make it in total `noopener noreferrer`. Pretty cool that the build was catching this!

- [ ] Add `noopener`

File: src/components/Activity.js [hunk 4]
```diff
@@ -1,20 +1,20 @@
 import React, { useEffect, useState } from "react";
 import Typography from "@material-ui/core/Typography";
 import MenuItem from "@material-ui/core/MenuItem";
-import Chip from "@material-ui/core/Chip";
+// import Chip from "@material-ui/core/Chip";
```
Comment (electrovir) [inline src/components/Activity.js:4]: Let's just delete all these unused lines of code in `Activity.js`

- [ ] delete unused lines

File: src/components/BuyForm.js [hunk 5]
```diff
@@ -39,7 +40,7 @@ export default function BuyForm(props) {
               <strong><span style={{fontSize:"1.5em",color:"red"}}>{props.price ? <PriceICP size={35} price={props.price} /> : "" }</span></strong><br />{props.price ? <>~<PriceUSD price={EntrepotGetICPUSD(props.price)} /></> : ""}
             </Grid>
           </Grid>
-          <small>This process may take a minute to process. Transactions can not be reversed. By clicking confirm you show acceptance of our <a href="https://docs.google.com/document/d/13aj8of_UXdByGoFdMEbbIyltXMn0TXHiUie2jO-qnNk/edit" target="_blank">Terms of Service</a></small>
+          <small>This process may take a minute to process. Transactions can not be reversed. By clicking confirm you show acceptance of our <a href="https://docs.google.com/document/d/13aj8of_UXdByGoFdMEbbIyltXMn0TXHiUie2jO-qnNk/edit" target="_blank" rel="noreferrer">Terms of Service</a></small>
```
Comment (electrovir) [inline src/components/BuyForm.js:43]: - [ ] `noopener noreferrer` again

File: src/components/Collection.js [hunk 6]
```diff
@@ -197,31 +197,36 @@ export default function Collection(props) {
   };
   const refresh = async (_sort, _collectionFilter) => {
     const _api = extjs.connect("https://boundary.ic0.app/", props.identity);
+    var r
```
Comment (electrovir) [inline src/components/Collection.js:200]: Can these be `let` instead of `var`?

- [ ] prefer `let`

File: src/components/Collection.js [hunk 7]
```diff
@@ -245,24 +250,26 @@ export default function Collection(props) {
...
-      var _sort = s ?? sort;
+      // var _sort = s ?? sort;
       var _collectionFilter = cf ?? collectionFilter;
       if (!_nfts) return;
       if (l) setNfts(l);
       var _displayNfts = _nfts;
-      _displayNfts = _displayNfts.filter((token,i) => (_collectionFilter == 'all' || extjs.decodeTokenId(token).canister == _collectionFilter));
+      _displayNfts = _displayNfts.filter((token,i) => (_collectionFilter === 'all' || extjs.decodeTokenId(token).canister === _collectionFilter));
       _displayNfts = _displayNfts.sort((a,b) => {
+        var bp
```
Comment (electrovir) [inline src/components/Collection.js:260]: - [ ] `let`, if possible

File: src/components/Collection.js [hunk 8]
```diff
@@ -1,7 +1,7 @@
-/* global BigInt */
+// /* global BigInt */
 import React from 'react';
 import Typography from '@material-ui/core/Typography';
-import Button from '@material-ui/core/Button';
+// import Button from '@material-ui/core/Button';
```
Comment (electrovir) [inline src/components/Collection.js:4]: - [ ] delete unused lines in `Collection.js`

File: src/components/BuyForm.js [hunk 9]
```diff
@@ -9,8 +9,9 @@ import Grid from '@material-ui/core/Grid';
 import Skeleton from '@material-ui/lab/Skeleton';
 import PriceICP from './PriceICP';
 import PriceUSD from './PriceUSD';
-import extjs from "../ic/extjs.js";
-import {EntrepotNFTImage, EntrepotDisplayNFT, EntrepotGetICPUSD} from '../utils.js';
+// import extjs from "../ic/extjs.js";
```
Comment (electrovir) [inline src/components/BuyForm.js:12]: - [ ] delete unused lines in `BuyForm.js`

**Review by electrovir — CHANGES_REQUESTED** [review]

**Comment by Ry-Hebert** [comment]
Thanks for the feedback on those. Also I was already scheduled with my other job last week but starting tomorrow I'm looking forward to having more time to spend on the project!

File: src/components/Collection.js (hunk 6 above)
Comment (Ry-Hebert) [reply src/components/Collection.js:200]: absolutely!

File: src/components/Activity.js (hunk 4 above)
Comment (electrovir) [reply src/components/Activity.js]: Those look meaningful to me, like they're supposed to be able to toggled back on quickly, so let's leave those for now.

**Comment by electrovir** [comment]
stale

**Pull request closed without merging**